perf/istio-install/create_cluster.sh

```diff
@@ -51,40 +53,39 @@ MAXPODS_PER_NODE=100
 # Labels and version
 ISTIO_VERSION=${ISTIO_VERSION:?"Istio version label"}
 DEFAULT_GKE_VERSION=$(default_cluster "${ZONE}")
+# shellcheck disable=SC2181
 if [[ $? -ne 0 ]];then
   echo "${DEFAULT_GKE_VERSION}"
   exit 1
 fi
 GKE_VERSION=${GKE_VERSION-${DEFAULT_GKE_VERSION}}
 
-# default scope for reference
-SCOPES_DEFAULT="https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append"
-
 # Full scope is needed for the context graph API
 SCOPES_FULL="https://www.googleapis.com/auth/cloud-platform"
 
 SCOPES="${SCOPES_FULL}"
 
 # A label cannot have "." in it.
-ISTIO_VERSION=$(echo ${ISTIO_VERSION} | sed 's/\./_/g')
+# shellcheck disable=SC2001
+ISTIO_VERSION=$(echo "${ISTIO_VERSION}" | sed 's/\./_/g')
 
 function gc() {
-  echo $*
+  echo "$*"
 
-  if [[ ! -z "${DRY_RUN}" ]];then
+  if [[ -n "${DRY_RUN}" ]];then
     return
   fi
 
-  gcloud $*
+  gcloud "$*"
```

Comment:
isnt this not the same? one is passing multiple values and one passes a single value? Or maybe that is $@... I hate bash

Reply:
It should continue to be $*

“$*” is incorrect .
Looks like shell linter does not understand intent.

Reply:
Yep, I had to write a program to prove it to myself. I'll revert these changes and add a warning suppression instead.